addons/mail/static/src/components/rtc_option_list/rtc_option_list.xml

```diff
@@ -24,7 +24,7 @@
                     <i class="o_RtcOptionList_buttonIcon fa fa-lg fa-cog"/>
                     <span class="o_RtcOptionList_buttonText">Settings</span>
                 </button>
-                <t t-if="env.isDebug()">
+                <t t-if="rtcOptionList.modelManager.isDebug">
```

Comment:
Reference to model manager from any record is a tad too technical for my taste.
I don't mind internal code of model to rely on this, but I prefer business code to avoid relying on that.
At the very least `messaging.modelManager.isDebug` could pass (`isDebug` is like a technical field of the global environment).

Reply:
Ok to access it from messaging, but it should stay on model manager because if it was an actual field it would be too costly to observe in other situations.